force-app/main/default/classes/GrossPickingTest.cls

```diff
@@ -0,0 +1,80 @@
+@IsTest
+private class GrossPickingTest {
```

Comment:
In both methods you have the same test data, at least most of them, you can create a method that will remove this duplication and use in both methods